tests/io/test_fsspec.py

```diff
@@ -61,7 +62,7 @@ def test_fsspec_new_input_file(fsspec_fileio: FsspecFileIO) -> None:
     assert input_file.location == f"s3://warehouse/{filename}"
 
 
-@pytest.mark.s3
+@pytest.mark.s3fsspec_file_io
```

Comment:
This seems to be an unrelated change

Reply:
Ah good catch, I think this was a copy/paste bug (somehow pasted fsspec_file_io on this line by mistake)